docs/os/security.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Security considerations
+parent: Operating System
+nav_order: 330
+---
+## Security considerations
+
+As a networked device, reachable from the outside, the attack surface of the BitBox Base needs to be minimized as much as possible. This is why we use our BitBox App as the dedicated user interface for node management. By default, only one port is exposed on the Base to communicate over an end-to-end encrypted channel with the App, providing all functionality.
+
+### Pairing Base with BitBox App
+The first step is to safely pair the BitBox Base and App. To achieve this, the Base announces itself on the local network using mDNS and the Middlware exposes provides an API endpoint to initiate a secure connection. The Base is automatically detected by the App and announced within the user interface. 
+
+At first, only a single public API endpoint for pairing is available that allows the two components to establish an encrypted communication channel. To rule out a man-in-the-middle attack, a confirmation code is shown both within the App and on the Base and needs to be confirmed manually. With the secure connection established, the [Noise encryption](http://noiseprotocol.org) can be set up and the full API functionality is available to the App.
+
+### Networking
+The following components help harden the Base against networking attacks:
+
+* **No web interface**: Modern web applications, built conveniently with frameworks like Node.js or Flask, have hundreds of dependencies that are outside our control and could contain security vulnerabilities or even malicious code. This is why we decided not to provide a web interface directly on the Base and expose only a single API by default.
```

Comment:
Perhaps when we say "No web interface" we might want to clarify that we're talking about HTTP servers? (For many people, "web" is synonymous with networked applications overall, and we do of course have networked applications, even if we do not have HTTP servers running on the Base.)

Reply:
I renamed it to "no browser interface"

Reply:
> I renamed it to "no browser interface"

I'm still seeing `No web interface` as of 7d614a343bdb92a76a73c9244e94b23e536227aa..